playwright.config.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint sort-keys: "off" */
+
+import { defineConfig, devices } from "@playwright/test";
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+require("dotenv").config();
+
+const PORT = process.env.PORT || 3000;
+const BASE_URL =
+  process.env.BASE_URL || `https://devbox.library.northwestern.edu:${PORT}`;
+
+export default defineConfig({
+  testDir: "./tests",
+  fullyParallel: true,
+
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  reporter: "html",
+  use: {
```

Comment:
Up the timeout to 10s.

```suggestion
  timeout: 10000,
  use: {
```